kafka-config.yml.sample

```diff
@@ -1,40 +1,69 @@
-- name: nri-kafka
-  env:
-    METRICS: "false"
-    INVENTORY: "false"
-    CONSUMER_OFFSET: "true"
-    CLUSTER_NAME: "testcluster3"
+    # Use bootstrap broker for discovery. The integration will connect to a fixed broker, described below, and retrieve
+    # all other brokers in the cluster from it.
     AUTODISCOVER_STRATEGY: "bootstrap"
+
+    # Bootstrap broker arguments. These configure a connection to a single broker. The rest of the brokers in the cluster
+    # will be discovered using that connection.
     BOOTSTRAP_BROKER_HOST: "localhost"
     BOOTSTRAP_BROKER_KAFKA_PORT: 9092
-    BOOTSTRAP_BROKER_KAFKA_PROTOCOL: PLAINTEXT
-    # A regex pattern that matches the consumer groups to collect metrics from
-    CONSUMER_GROUP_REGEX: '.*'
+    BOOTSTRAP_BROKER_KAFKA_PROTOCOL: PLAINTEXT # Currently support PLAINTEXT and SSL
+
+    # When bootstrap broker is used for discovery, JMX credentials and port must be the same for all brokers.
+    BOOTSTRAP_BROKER_JMX_PORT: 9999
+    BOOTSTRAP_BROKER_JMX_USER: admin
+    BOOTSTRAP_BROKER_JMX_PASSWORD: password
+
+    # Setting this to true will make the integration retrieve data only from the broker configured as bootstrap,
+    # and no other brokers will be discovered from it.
+    # Users will typically want to set this to false (default), unless a different discovery mechanism, such as Docker
+    # or Kubernetes, is used on top of the integration.
+    LOCAL_ONLY_COLLECTION: false
+
+    # This field is used to toggle the collection of broker and topic metrics. This is on by default and should only be set to "false"
+    # for the specific case where only producers/consumers are being monitored and "topic_mode" is set to "All".
+    # Any other case this field can be omitted.
+    COLLECT_BROKER_TOPIC_DATA: true
+
+    # Below are the fields used to fine tune/toggle topic metric collection. Can be one of:
+    # - "all", where all topics found in all brokers will be collected.
+    # - "none", which will disable topic collection alltogether.
+    # - "list", which allows a fixed list of topics to be defined as topic_list: '["topic1", "topic2"]'
+    # - "regex", which allows to define a regex to select which topics will be monitored using topic_regex: 'topic\d+'
+    #
+    # TOPIC_MODE: "all"
+    # TOPIC_MODE: "none"
+    TOPIC_MODE: "regex"
+    TOPIC_REGEX: 'topic\d+'
+    # TOPIC_MODE: "list"
+    # TOPIC_LIST: `["topic1", "topic2", "topic3"]`
+
+    # Topic size and offset can also be collected from brokers in the cluster. This operations are intensive and can take a while
+    # to collect for a larger number of topics. It is recommended to only enable this features when collecting topics
+    # selectively, i.e. topic_mode is not "all" or a very permissive regex.
+    # If the fields are omitted they will default to false.
+    COLLECT_TOPIC_SIZE: false
+    COLLECT_TOPIC_OFFSET: false
+
+    METRICS: "true"
   interval: 15s
+  inventory_source: config/kafka
```

Comment:
Should we add this is. a metrics only config?

Reply:
yes sure, we can remove it, I will do it before merging